endpoints/openrtb2/auction_test.go

```diff
@@ -1686,11 +1826,26 @@ func (e *mockBidExchange) HoldAuction(ctx context.Context, bidRequest *openrtb.B
 				return nil, err
 			}
 
+			if rawPrebidExt, ok := bidderExts[openrtb_ext.PrebidExtKey]; ok {
+				var prebidExt openrtb_ext.ExtImpPrebid
+				if err := json.Unmarshal(rawPrebidExt, &prebidExt); err == nil && prebidExt.Bidder != nil {
+					for bidder, ext := range prebidExt.Bidder {
+						if ext == nil {
+							continue
+						}
+
+						bidderExts[bidder] = ext
+					}
+				}
+			}
+
```

Comment:
The mock `HoldAuction` was expanded to behave more like the real one. In the future, using the real `HoldAuction` with significant mock bidders and test objects should be the go to.